pkg/controller/kfservice/resources/knative/configuration.go

```diff
@@ -126,13 +129,13 @@ func (c *ConfigurationBuilder) CreateKnativeConfiguration(name string, metadata
 
 func configurationAnnotationFilter(annotationKey string) bool {
 	switch annotationKey {
-	case autoscaling.TargetAnnotationKey:
-		return true
-	case autoscaling.ClassAnnotationKey:
-		return true
-	case constants.KFServiceGKEAcceleratorAnnotationKey:
-		return true
-	default:
+	case autoscaling.MinScaleAnnotationKey:
```

Comment:
filter min/max scale annotations because we already have those fields on the kfservice spec?

Reply:
Yes, that's right.